pkg/services/sqlstore/migrations/libraryelements.go

```diff
@@ -61,4 +61,11 @@ func addLibraryElementsMigrations(mg *migrator.Migrator) {
 
 	mg.AddMigration("alter library_element model to mediumtext", migrator.NewRawSQLMigration("").
 		Mysql("ALTER TABLE library_element MODIFY model MEDIUMTEXT NOT NULL;"))
+
+	mg.AddMigration("add library_element folder uid", migrator.NewAddColumnMigration(libraryElementsV1, &migrator.Column{
+		Name: "folder_uid", Type: migrator.DB_NVarchar, Length: 40, Nullable: false, Default: "''",
```

Comment:
Is it possible to create it with `Nullable: true`, then populate it with the update and in the end, change the property to `Nullable: false`?
I think we don't want to have a blank default value on this column but restrict it not to allow empty values.

Reply:
this migration is the same as in the dashboard table, would like to keep it the same

Reply:
in the [dashboard table](https://github.com/grafana/grafana/blob/5de17432f544256bf44cfae6495104fcd0a9b3c4/pkg/services/dashboards/database/migrations/folder_uid_migrator.go#L73) the `folder_uid` column is nullable